File: Bubble_Sort/bubble_sort.js
const bubble_sort = (seq) => {
    for (i = 0; i < seq.length; i++) {
        for (j = 1; j < seq.length; j++) {
            if (seq[ j ] < seq[ j - 1 ]) {
                let temp = seq[ j ];
                seq[ j ] = seq[ j - 1 ];
                seq[ j - 1 ] = temp;
            }
        }
    }
};

let my_list = [ 1, 8, 5, 2, 0 ];
bubble_sort(my_list);
console.log(my_list);

// Big-O = O(n^2)

File: Algorithm/Bubble_Sort/bubble_sort.py
def bubble_sort(seq):
    for i in range(len(seq)):
        for j in range(len(seq)-i-1):
            if(seq[j] > seq[j+1]):
                seq[j], seq[j+1] = seq[j+1], seq[j]

myList = [1,8,5,2,0]
print(myList)
bubble_sort(myList)
print(myList)